src/ceph.rs

```diff
@@ -326,6 +327,7 @@ pub fn disconnect_from_ceph(cluster: rados_t) {
 }
 
 /// Set the value of a configuration option
+/// NOTE: the rados_t parameter must not be connected to ceph yet when this is called.
```

Comment:
How does that work?! And why is this one different than every other one?

Reply:
Great question.  I had to jump on the ceph-devel irc channel to talk to them about this.  This function doesn't do what I thought it did.  I thought it was setting configuration options in the ceph.conf file.  I was wrong.  What it actually does it set configuration options on the rados connection to ceph.  Because it's potentially setting things that could effect the rados connection they require that you not be connected when running this.  The library doesn't support this mode of operation yet so I put a note there for future confused people.